README.md

```diff
@@ -0,0 +1,74 @@
+# Console for Apache Kafka™ on Kubernetes
+
+This project is a web console designed to facilitate interactions with Apache Kafka™ instances on Kubernetes, leveraging the [Strimzi](https://strimzi.io) Cluster Operator. 
+It is composed of two main parts: 
+
+- a REST API backend developed with Java Quarkus
+- a user interface (UI) built with Next.js and  [PatternFly](https://patternfly.org)
+
+### Getting started
+
+#### API
+
+Ensure you have Kubernetes and Strimzi Cluster Operator installed on your system.
+
+```bash
+cd api
```

Comment:
are you sure that we want the user to run everything in "dev mode"?
I guess, it would be easier to run things as containers in the cluster, and reserve those instructions for a `Contributing` document.